tests/test_kiwi.py

```diff
@@ -55,7 +55,10 @@ def test_kiwi_installation(auto_container):
     reason="test is atm x86_64 specific",
 )
 @pytest.mark.skipif(
-    PODMAN_SELECTED and os.geteuid() != 0,
+    PODMAN_SELECTED, reason="Kiwi containers are only supported with podman"
```

Comment:
shouldn't that be `DOCKER_SELECTED` then? otherwise the reason does not match the condition.

Reply:
or `not PODMAN_SELECTED`

Reply:
My brain is not braining anymore :melting_face: